.github/workflows/alert-deploy-arena-adapter.yaml

```diff
@@ -0,0 +1,38 @@
+name: Deploy alerts to NAIS for Mulighetsrommet-arena-adapter
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - alerts-arena-adapter.yaml
```

Comment:
Denne linja kan vel fjernes til fordel for neste linje?

(Det samme gjelder i `.github/workflows/alert-deploy-api.yaml`)

Reply:
Ja, godt poeng. Jeg fjerner fra begge filene.